tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/lists/ArrayDestructureTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.test.types.lists;
```

Comment:
I think this test belong here https://github.com/ballerina-platform/ballerina-lang/tree/e8ad87cc9c315cd9892905ecd5eacf49018efb4c/tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/varref

Reply:
fixed with 60bfc55